### PR TITLE
III-5685 Remove `CreateCopy` command and `CopyCreated` event

### PR DESCRIPTION
**Pull request by LucWollants**
### Removed
- Removed command `CreateCopy`
- Removed event `CopyCreated`
- Removed the now obsolete `parentUuid`

---
Ticket: https://jira.uitdatabank.be/browse/III-5685

### CODE REVIEW

File: tests/Label/ReadModels/JSON/Repository/EntityTest.php [hunk 1]
```diff
@@ -181,7 +139,6 @@ public function it_requires_a_positive_count(): void
             $this->name,
             $this->visibilty,
             $this->privacy,
-            $this->parentUuid,
```
Comment (JonasVHG) [inline tests/Label/ReadModels/JSON/Repository/EntityTest.php:184]: Is this test still needed if `count` will be removed?

**Review by JonasVHG — APPROVED** [review]
Maybe best make a subticket for the migration to remove the parentUuid, because a deploy to production might (not) happen soon.

File: tests/Label/ReadModels/JSON/Repository/EntityTest.php (hunk 1 above)
Comment (LucWollants) [reply tests/Label/ReadModels/JSON/Repository/EntityTest.php:184]: Indeed, but that is done here https://github.com/cultuurnet/udb3-backend/pull/1395

**Comment by LucWollants** [comment]
> Maybe best make a subticket for the migration to remove the parentUuid, because a deploy to production might (not) happen soon.

I created https://jira.uitdatabank.be/browse/III-5689 to fix this.

**Pull request merged**